docs/how_to/nginx_https.md

```diff
@@ -0,0 +1,49 @@
+# How to use Nginx to serve Livebook over HTTPS
+
+This guide shows you how to use Nginx to serve Livebook over HTTPS.
+
+## Prerequisites
+
+- Nginx installed
+- Livebook installed
+- Both Livebook and Nginx running on the same machine or within the same network
+- SSL certificate and key files
+
+## Nginx configuration
+
+Use the following Nginx config file as a starting point:
+
+```nginx
+http {
```

Comment:
For some reason, the version of `nginx` I am using (in container `image: nginx:latest`) did not like the `http` outer context. But this worked for you @hugobarauna so just really leaving this as a recorded comment!

Reply:
I'm using the version from the `stable` tag from the Nginx docker image, which I think is Nginx 1.26.1.